torchmetrics/text/bert.py

```diff
@@ -115,18 +117,18 @@ class BERTScore(Metric):
         >>> from torchmetrics.text.bert import BERTScore
         >>> preds = ["hello there", "general kenobi"]
         >>> target = ["hello there", "master kenobi"]
-        >>> bertscore = BERTScore()
-        >>> bertscore(preds, target)  # doctest: +SKIP
-        {'precision': [0.99..., 0.99...],
-         'recall': [0.99..., 0.99...],
-         'f1': [0.99..., 0.99...]}
+        >>> bert = BERTScore(model_name_or_path="roberta-base")
+        >>> bert(preds, target)  # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        {'precision': [0.999..., 0.999., 0.987..., 0.987...],
+         'recall': [0.999..., 0.999..., 0.987..., 0.987...],
+         'f1': [0.999..., 0.999..., 0.987..., 0.987...]}
```

Comment:
This looks quite odd to me as the length of output should be 2 instead of 4.. gonna have a look

Reply:
so it is good that we have these examples tested again... :rabbit: